tests/capabilities.js

```diff
@@ -1,21 +1,11 @@
-(function() {
-var a = require('assert');
-var _ = require('underscore');
-var assert = a.assert;
-var eq_ = a.eq_;
-var eeq_ = a.eeq_;
-var feq_ = a.feq_;
-var mock = a.mock;
-var capabilities = require('capabilities');
-
-test('capabilities device_platform', function(done, fail) {
-    mock(
-        'capabilities',
-        {
-            settings: {
-            }
-        },
-        function (capabilities) {
+define(['assert', 'capabilities'], function(a, capabilities) {
+    var assert = a.assert;
```

Comment:
Agreed. Once I get all the tests running in karma/mocha I'll switch these over to use chai.